pallets/circuit/src/lib.rs

```diff
@@ -676,14 +661,13 @@ pub mod pallet {
         pub fn on_extrinsic_trigger(
             origin: OriginFor<T>,
             side_effects: Vec<SideEffect<T::AccountId, EscrowedBalanceOf<T, T::Escrowed>>>,
-            fee: EscrowedBalanceOf<T, T::Escrowed>,
             sequential: bool,
```

Comment:
any reasons why we're still keeping this one?

Reply:
could be useful from 3vm to enforce steps order - feature needs adding but the interface can stay imo